tests/api/console/test_cli_console_output.py

```diff
@@ -91,18 +100,35 @@ def test_phase_is_cleaned_up_on_exception(cli_console):
         pass
 
 
-def test_phase_cannot_be_indented(cli_console):
+def test_phase_inside_indented(cli_console, capsys):
+    cli_console.step("Outside of Indent")
     with cli_console.indented():
-        with pytest.raises(CliConsoleNestingProhibitedError):
-            with cli_console.phase("Enter"):
-                pass
+        with cli_console.phase("Enter"):
```

Comment:
the check is too weak IMHO. Add a step above to capture that phase + step are indented to the same level, and something inside the phase to capture that it indents _on top of_ the existing indented block